cmd/controller/controller.go

```diff
@@ -170,7 +169,7 @@ func (c *CmdOpts) startController() error {
 			LogLevel:    c.Logging["etcd"],
 		}
 	default:
-		return errors.New(fmt.Sprintf("Invalid storage type: %s", c.ClusterConfig.Spec.Storage.Type))
+		return fmt.Errorf("Invalid storage type: %s", c.ClusterConfig.Spec.Storage.Type)
```

Comment:
I don't actually understand how the linter pass that, the errors should start with lowercase :thinking: